### ISSUE TITLE
FxaMigrationException: Failed to sign-into authenticated account in state Cohabiting

### ISSUE THREAD

**Issue by pocmo** [mozilla-mobile/android-components#5693]
https://sentry.prod.mozaws.net/operations/fenix-fennec-nightly/issues/7066001/

```
FxaMigrationException: Failed to sign-into authenticated account in state Cohabiting
    at mozilla.components.support.migration.AuthenticatedAccountProcessor.signIn$support_migration_release(FennecFxaMigration.kt:22)
    at mozilla.components.support.migration.AuthenticatedAccountProcessor$signIn$1.invokeSuspend
    at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:2)
    at kotlinx.coroutines.DispatchedTask.run(DispatchedTask.kt:15)
    at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1167)
...
(2 additional frame(s) were not displayed)

[INFO] Failed to sign-into authenticated account in state Cohabiting
```

**Comment by pocmo** [comment]
Not an actual crash!

**Comment by pocmo** [comment]
Sentry issue: [FENIX-FENNEC-NIGHTLY-1H](https://sentry.prod.mozaws.net/operations/fenix-fennec-nightly/issues/7137149/)

**Comment by pocmo** [comment]
A similar issue: https://sentry.prod.mozaws.net/operations/fenix-fennec-nightly/issues/7137149
But state `Married`.

```
mozilla.components.support.migration.FxaMigrationException: Failed to sign-into authenticated account in state Married
    at mozilla.components.support.migration.AuthenticatedAccountProcessor.signIn$support_migration_release(FennecFxaMigration.kt:22)
    at mozilla.components.support.migration.AuthenticatedAccountProcessor$signIn$1.invokeSuspend
    at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:2)
    at kotlinx.coroutines.DispatchedTask.run(DispatchedTask.kt:15)
    at java.util.concurrent.ThreadPoolExecutor.processTask(ThreadPoolExecutor.java:1187)
    at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1152)
    at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:641)
    at java.lang.Thread.run(Thread.java:784)
```

**Comment by grigoryk** [comment]
This should be covered by https://github.com/mozilla-mobile/android-components/pull/5800

**Comment by grigoryk** [comment]
Marking this as closed: in #5800 we shipped support for migrating FxA while device is offline (airplane mode, or just poor network connectivity). Volume for this exception should go down significantly. It is still possible for us to see it - for example, if we encountered a "non-recoverable" type of an error, e.g. bad session token, or certain types of server errors. However, in those cases there's nothing we can do other than track volume of such events.
